app/views/users/show.html.erb

```diff
@@ -14,6 +14,7 @@
     <h2>Displaying <%= params[:filter].nil? ? 'all' : params[:filter] %> predictions made by <%= show_user(@user) %>. Filter by:
       <%= form_tag @user, method: :get, class: 'prediction-filter' do %>
         <%= select_tag :filter, options_for_select(['all predictions', 'judged', 'unjudged', 'future'], params[:filter]) %>
+        <%= select_tag :tags, options_for_select(tag_name_options, params[:tags]), prompt: "no tags" %>
```

Comment:
This could be updated to allow multi-select without too much effort, since the query object itself can already handle multiple tag names properly.

Reply:
Why wouldn't we just return a list of the tags that the user actually has predictions for? Surely that's both simpler, more scalable and more user-friendly?

Reply:
Also applies to https://github.com/bellroy/predictionbook/pull/203/files#r794229948

Reply:
I added the more future proof version [here](https://github.com/bellroy/predictionbook/pull/203/commits/97909a5172b959aefa0b2a15a737a97e7cf708f1).